README.md

```diff
@@ -63,12 +63,12 @@ function getUserPosts(id, blogId, limit, offset) {
 }
 ~~~
 
-As you can see, this minimal example is already rather hard to read. It is also incorrect:
+As we can see, this minimal example is already rather hard to read. It is also incorrect:
 
-- I forgot that there was a trailing slash at the end of the `API_URL` constant so the slash got duplicated (`https://api.example.com//users`)
+- One common issue is of trailing slashes, `API_URL` has a trailing slash in the example. This will lead to URL containing duplicated slashes (`https://api.example.com//users`).
```

Comment:
I agree that the phrase "the slash got duplicated" sounds unnatural. Can we just keep the second part ("This will lead to URL containing duplicated slashes") from here?